### PR TITLE
Initial release notes upload

### PR DESCRIPTION
**Pull request by alevsh**

### CODE REVIEW

File: net/release-notes/release-notes-2021/aspose-pdf-for-net-21-7-release-notes/_index.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+This page contains release notes information for Aspose.PDF for .NET 21.7.
+
+{{% /alert %}} 
+
+## Improvements and Changes
+
+|**Key**|**Summary**|**Category**|
+| :- | :- | :- |
+|PDFNET-49595 Add memory optimized stream to load huge files using Stream based constructor|Feature|
```
Comment (andruhovski) [inline net/release-notes/release-notes-2021/aspose-pdf-for-net-21-7-release-notes/_index.md:24]: Advanced support of Streams / Advanced handling huge documents

File: net/release-notes/release-notes-2021/aspose-pdf-for-net-21-7-release-notes/_index.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+|PDFNET-50068 PDF creation with XML and XLS with parameters|Feature|
```
Comment (andruhovski) [inline net/release-notes/release-notes-2021/aspose-pdf-for-net-21-7-release-notes/_index.md:25]:
```suggestion
|PDFNET-50068 PDF creation based on XML and XSL with parameters|Feature|
```

**Review by andruhovski — CHANGES_REQUESTED** [review]

File: net/release-notes/release-notes-2021/aspose-pdf-for-net-21-7-release-notes/_index.md (hunk 1 above)
Comment (alevsh) [reply net/release-notes/release-notes-2021/aspose-pdf-for-net-21-7-release-notes/_index.md:24]: Advanced streams support to handle huge documents?

File: net/release-notes/release-notes-2021/aspose-pdf-for-net-21-7-release-notes/_index.md (hunk 2 above)
Comment (alevsh) [reply net/release-notes/release-notes-2021/aspose-pdf-for-net-21-7-release-notes/_index.md]: Fixed

**Pull request merged**